lib/screens/homePage/home.dart

```diff
@@ -15,97 +15,113 @@ class HomePage extends StatefulWidget {
-                        return Text(highestJump,
-                            style: Theme.of(context)
-                                .textTheme
-                                .headline4
-                                .copyWith(fontWeight: FontWeight.bold));
-                      },
-                    ),
-                  ],
-                ),
-                SizedBox(height: 32),
-                Text('Instructions',
-                    style: Theme.of(context)
-                        .textTheme
-                        .headline5
-                        .copyWith(fontWeight: FontWeight.bold)),
-                SizedBox(height: 16),
-                Text('1. Stand on the mat, align your feet to the pads',
-                    style: Theme.of(context).textTheme.subtitle1),
-                SizedBox(height: 32),
-                Text(
-                    '2. When you press JUMP we will start counting down form 3, jump on GO.',
-                    style: Theme.of(context).textTheme.subtitle1),
-                SizedBox(height: 32),
-                Text('3. Try to land with both feet on the pads.',
-                    style: Theme.of(context).textTheme.subtitle1),
-              ],
+                          return Text(highestJump,
+                              style: Theme.of(context)
+                                  .textTheme
+                                  .headline4
+                                  .copyWith(fontWeight: FontWeight.bold));
+                        },
+                      ),
+                    ],
+                  ),
+                  SizedBox(height: 32),
+                  Text('Instructions',
+                      style: Theme.of(context)
+                          .textTheme
+                          .headline5
+                          .copyWith(fontWeight: FontWeight.bold)),
+                  SizedBox(height: 16),
+                  Text('1. Stand on the mat, align your feet to the pads',
+                      style: Theme.of(context).textTheme.subtitle1),
+                  SizedBox(height: 32),
+                  Text(
+                      '2. When you press JUMP we will start counting down form 3, jump on GO.',
+                      style: Theme.of(context).textTheme.subtitle1),
+                  SizedBox(height: 32),
+                  Text('3. Try to land with both feet on the pads.',
+                      style: Theme.of(context).textTheme.subtitle1),
+                ],
+              ),
             ),
           ),
         ),
+        floatingActionButton: _isVisible ? button1() : button2(),
       ),
-      floatingActionButton: FloatingActionButton.extended(
-          onPressed: () {
-             Navigator.pushNamed(context, "/standonmat",
+    );
+  }
+
+  FloatingActionButton button1() {
+    return FloatingActionButton.extended(
+        onPressed: () {
+          Navigator.pushNamed(context, "/standonmat",
               arguments: Jump(DateTime.now(), 177, 4));
```

Comment:
I don't think you have to pass the jump as an argument as this gave an error last time correct me if i'm wrong